common/src/main/java/org/apache/falcon/metadata/MetadataMappingService.java

```diff
@@ -370,4 +373,8 @@ private void updateImportedFeedInstance(WorkflowExecutionContext context) throws
         LOG.info("Updating imported feed instance: {}", context.getNominalTimeAsISO8601());
         instanceGraphBuilder.addImportedInstance(context);
     }
+    private void updateExportedFeedInstance(WorkflowExecutionContext context) throws FalconException {
+        LOG.info("Adding export feed instance: {}", context.getNominalTimeAsISO8601());
```

Comment:
Better to use "Updating" instead of "Adding" for consistency. Also, this will be called to either add new feed instance or update the instance status.